.github/workflows/ci.yml

```diff
@@ -141,17 +141,17 @@ jobs:
 
       - name: Install lowest dependencies with composer
         if: matrix.dependencies == 'lowest'
-        run: ./tools/composer update --no-ansi --no-interaction --no-progress --prefer-lowest
+        run: php ./tools/composer update --no-ansi --no-interaction --no-progress --prefer-lowest
```

Comment:
@shivammathur

Do you have an idea why it would be necessary to specify the `php` binary here for running these commands on `windows-latest`?

Without it, they are not run, see #4333.

Reply:
@localheinz 
`#!/usr/bin/env php` does not work on Windows, so you will have to call local `phar` tools with `php`. 

When tools are installed with `setup-php` on Windows, it creates a `batch` script to call the tools with php.

Reply:
Thank you, @shivammathur!